oryx-app-mllib/src/test/java/com/cloudera/oryx/app/mllib/kmeans/AbstractKMeansEvalIT.java

```diff
@@ -27,17 +32,17 @@
 import com.cloudera.oryx.app.kmeans.KMeansPMMLUtils;
 import com.cloudera.oryx.app.kmeans.KMeansPMMLUtilsTest;
 import com.cloudera.oryx.app.kmeans.SquaredDistanceFn;
-import com.cloudera.oryx.common.OryxTest;
+import com.cloudera.oryx.lambda.AbstractSparkIT;
 
-public abstract class AbstractKMeansEvalTest extends OryxTest {
+public abstract class AbstractKMeansEvalIT extends AbstractSparkIT implements Serializable {
```

Comment:
The test should not have to implement `Serializable` or really we should fix up the code so the test object is not in a closure.